### PR TITLE
=pro bintray jcentral resolver for jenkins-ci deps

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
Merging to avoid unresolved deps on `master`.

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,6 +2,10 @@ resolvers += Classpaths.typesafeResolver
 
 resolvers += "jgit-repo" at "http://download.eclipse.org/jgit/maven"
 
+// need this to resolve http://jcenter.bintray.com/org/jenkins-ci/jenkins/1.26/
+// which is used by plugin "org.kohsuke" % "github-api" % "1.68"
+resolvers += "Bintray Jcenter" at "http://jcenter.bintray.com"
```
Comment (hseeberger) [inline project/plugins.sbt]: Why not `resolvers += Resolver.jcenterRepo`?

Comment (ktoso) [inline project/plugins.sbt]: Interesting, intellij didn't autocomplete this one for me; Sure, will update

Comment (ktoso) [inline project/plugins.sbt]: PR https://github.com/akka/akka/pull/17450

Comment (ktoso) [inline project/plugins.sbt]: Thanks! PR #17450

Comment (hseeberger) [inline project/plugins.sbt]: You're welcome!

**Pull request merged**